### PR TITLE
add unfurling to home page

### PR DESCRIPTION
**Pull request by stushurik**

### CODE REVIEW

File: packages/controllers/src/serveVizPage.js [hunk 1]
```diff
@@ -1,30 +1,10 @@
-import path from 'path';
 import { GetVisualization } from 'vizhub-use-cases';
+import { servePage } from './servePage';
```
Comment (curran) [inline packages/controllers/src/serveVizPage.js:2]: @stushurik This file looks like accidentally not committed?

**Review by curran — CHANGES_REQUESTED** [review]

File: packages/controllers/src/serveVizPage.js (hunk 1 above)
Comment (stushurik) [reply packages/controllers/src/serveVizPage.js:2]: @curran which one?

Comment (stushurik) [reply packages/controllers/src/serveVizPage.js:2]: the diff looks like I expect

Comment (curran) [reply packages/controllers/src/serveVizPage.js:2]: @stushurik I see no file in the repo called `servePage`. When I check out this branch locally and try to run, this is what I see:

image

Comment (stushurik) [reply packages/controllers/src/serveVizPage.js:2]: strange, looks like i didn't push one file

File: packages/controllers/src/servePage.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  <meta name="twitter:image" content="${image}" />
+  <meta name="twitter:domain" value="vizhub.com" />
+  <meta property="og:url" content="${url}"/>
+  <meta property="og:title" content="${title}" />
+  <meta property="og:description" content="${descriptionPlainText}" />
+  <meta property="og:image" content="${image}" />
+  <meta property="og:site_name" content="VizHub" />
+`;
+
+export const servePage = (indexHTML, { title, description, image, url}) => {
```
Comment (curran) [inline packages/controllers/src/servePage.js:22]: Ahhhhh there it is! Very nice.

**Review by curran — CHANGES_REQUESTED** [review]
Looks great!

**Review by curran — APPROVED** [review]
Great - everything is there. Thanks!

**Pull request merged**